tests/python/common/test_heterograph-apply-edges.py

```diff
@@ -260,6 +278,23 @@ def _print_error(a, b):
             _test(lhs, rhs, binary_op)
 
 
+# Here we test heterograph with only 1 source node because the format of node
+# feature is different from ordinary heterograph.
+@unittest.skipIf(
+    dgl.backend.backend_name != "pytorch", reason="Only support PyTorch for now"
+)
+@parametrize_idtype
+def test_heterograph_with_same_source_node_apply_edges(idtype):
+    hg = create_random_hetero_with_same_source_node(idtype)
+
+    hg.nodes["n1"].data["h"] = F.randn((hg.num_nodes("n1"), 1))
+    hg.nodes["n2"].data["h"] = F.randn((hg.num_nodes("n2"), 1))
+    hg.nodes["n3"].data["h"] = F.randn((hg.num_nodes("n3"), 1))
+
+    assert type(hg.srcdata["h"]) == F.Tensor
```

Comment:
`F.Tensor` is available? or we could just utilize `F.is_tensor()`

Reply:
OK，I changed to torch.Tensor here.